www/scripts/codecheckerviewer/filter/DateFilter.js

```diff
@@ -214,6 +215,25 @@ function (declare, Deferred, dom, DateTextBox, TimeTextBox, SelectFilter,
           this.setDayInterval(yesterday);
           this.selectedItem = value;
           break;
+        case 'this week':
```

Comment:
Isn't there a way to make this switch an integer-based one as opposed to string equality?